### PR TITLE
batch update and delete examples

### PR DESCRIPTION
**Pull request by Icemap**

### CODE REVIEW

**Comment by Icemap** [comment]
@Mini256  PTAL

File: plain-java-batch-delete/pom.xml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      <artifactId>mysql-connector-java</artifactId>
+      <version>8.0.28</version>
+    </dependency>
+
+    <dependency>
+      <groupId>com.zaxxer</groupId>
+      <artifactId>HikariCP</artifactId>
+      <version>5.0.1</version>
+    </dependency>
+  </dependencies>
```
Comment (Mini256) [inline plain-java-batch-delete/pom.xml:39]: 🤔 It seems that we have not used the connection pool.

File: plain-java-batch-delete/src/main/java/com/pingcap/bulkDelete/BatchDeleteExample.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        MysqlDataSource mysqlDataSource = new MysqlDataSource();
+
+        // Set server name, port, database name, username and password.
+        mysqlDataSource.setServerName("localhost");
+        mysqlDataSource.setPortNumber(4000);
+        mysqlDataSource.setDatabaseName("bookshop");
+        mysqlDataSource.setUser("root");
+        mysqlDataSource.setPassword("");
+
+        while (true) {
```
[the remark this answers is not in the document]
Comment (Icemap) [reply plain-java-batch-delete/src/main/java/com/pingcap/bulkDelete/BatchDeleteExample.java:41]: This example written for [bulk-update](https://docs.pingcap.com/zh/tidb/dev/update-data#Java%20(JDBC)) document. Need an infinity loop to update data.

File: plain-java-batch-delete/src/main/java/com/pingcap/bulkDelete/BatchDeleteExample.java [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+            String sql = "DELETE FROM `bookshop`.`ratings` WHERE `rated_at` >= ? AND  `rated_at` <= ? LIMIT 1000";
+            PreparedStatement preparedStatement = connection.prepareStatement(sql);
+            Calendar calendar = Calendar.getInstance();
+            calendar.set(Calendar.MILLISECOND, 0);
+
+            calendar.set(2022, Calendar.APRIL, 15, 0, 0, 0);
+            preparedStatement.setTimestamp(1, new Timestamp(calendar.getTimeInMillis()));
+
+            calendar.set(2022, Calendar.APRIL, 15, 0, 15, 0);
+            preparedStatement.setTimestamp(2, new Timestamp(calendar.getTimeInMillis()));
```
[the remark this answers is not in the document]
Comment (Icemap) [reply plain-java-batch-delete/src/main/java/com/pingcap/bulkDelete/BatchDeleteExample.java:58]: Ditto.

File: plain-java-batch-delete/pom.xml (hunk 1 above)
Comment (Icemap) [reply plain-java-batch-delete/pom.xml]: Yep. I removed it.

File: plain-java-batch-delete/src/main/java/com/pingcap/bulkDelete/BatchDeleteExample.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (Icemap) [reply plain-java-batch-delete/src/main/java/com/pingcap/bulkDelete/BatchDeleteExample.java:41]: Sorry. Should be [bulk-delete](https://docs.pingcap.com/zh/tidb/dev/delete-data#%E6%89%B9%E9%87%8F%E5%88%A0%E9%99%A4) document.

File: plain-java-batch-update/src/main/java/com/pingcap/bulkUpdate/BatchUpdateExample.java [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+        mysqlDataSource.setUser("root");
+        mysqlDataSource.setPassword("");
+
+        UpdateID lastID = batchUpdate(mysqlDataSource, null);
+
+        System.out.println("first time batch update success");
+        while (true) {
+            TimeUnit.SECONDS.sleep(1);
+            lastID = batchUpdate(mysqlDataSource, lastID);
+            System.out.println("batch update success, [lastID] " + lastID);
```
[the remark this answers is not in the document]
Comment (Icemap) [reply plain-java-batch-update/src/main/java/com/pingcap/bulkUpdate/BatchUpdateExample.java:78]: Nope. Data might be inserted at any time. So the loop should be infinity.

**Review by Mini256 — APPROVED** [review]

**Pull request merged**